.github/workflows/test.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           architecture: ${{matrix.architecture}}
 
       - name: Install Tox
-        run: pip install tox
+        run: pip install 'tox<4'
```

Comment:
I am ok to limit tox to < 4. I have had problems with tox 4 already in other projects, where I am actually considering dropping it.

Reply:
I certainly care more about this PR than tox 4 support.  `:]`  Since the other PR isn't actually working for both tox 3 and 4 I'd pick going with this 3-only option for now and 4 support can be figured out at some later time.  Maybe soon, maybe not.

Reply:
I am perfectly ok to drop #125 and leave this change here. I don't think tox 4 is mature yet, and tbh I hardly see the need of tox altogether (I'm dropping it from psycopg3 and it turns out it massively simplify the tests setup).